src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx

```diff
@@ -168,7 +170,10 @@ export class SavedObjectEdition extends Component<
       }
     );
     if (confirmed) {
+      const uiActions = this.props.serviceRegistry.get('uiActions')?.service as UiActionsStart;
```

Comment:
As ive called out in my other comment, this `uiActions` can simply be a prop thats passed down like `notifications`, but unlike `notifications` that comes from `core`, this will come from the start dependencies that you have already declared.